source/assets/stylesheets/styles/_project_story_large_tile.scss

```diff
@@ -0,0 +1,95 @@
+.project-story-large-tile {
+  box-sizing: border-box;
+  display: block;
+  margin-bottom: 6px;
+  padding-right: 6px;
+  text-decoration: none;
+
+  @include breakpoint($tablet-breakpoint) {
+    float: left;
+    margin-bottom: 0;
+    width: 66%;
+  }
+}
+
+.project-story-large-tile__container {
+  background-color: $project-story-tile__background-color;
+  box-sizing: border-box;
+  border: 1px solid $project-story-tile__border-color;
+  padding: 6px;
+  width: 100%;
+
+  &:hover {
+    background-color: $project-story-tile__hover-color;
+  }
+}
+
+.project-story-large-tile__image-container {
+  box-sizing: border-box;
+}
+
+.project-story-large-tile__image {
+  display: block;
+  height: 125px;
+  object-fit: cover;
+  width: 100%;
+
+  @include breakpoint($max-mobile-container) {
+    height: 180px;
+  }
+
+  @include breakpoint($tablet-breakpoint) {
+    height: 255px;
+  }
+
+  @include breakpoint($tablet-breakpoint) {
+    height: 276px;
+  }
+}
+
+.project-story-large-tile__content {
+  display: inline-block;
+  min-height: 60px;
+  padding-left: 8px;
+  padding-top: 10px;
+  text-decoration: none;
+
+  @include breakpoint($max-mobile-container) {
+    min-height: 75px;
+  }
+
+  @include breakpoint($tablet-breakpoint) {
+    padding: 20px 6px 0;
+    width: 85%;
+    min-height: 170px;
```

Comment:
For tablet upwards I can understand the requirement for `min-height` as we have a fixed layout. But for mobile we could be more relaxed and allow the size of the text to determine its height - just ensuring that we have some breathing space at the bottom of the tile! See below:

<img width="319" alt="screen shot 2015-11-19 at 10 13 24" src="https://cloud.githubusercontent.com/assets/885223/11267995/7835582e-8ea6-11e5-9ca8-3da65f0a603d.png">

(top needs more padding, bottom has too much space)